detect-nuget-inspector/detect-nuget-inspector/DependencyResolution/Project/ProjectXmlResolver.cs

```diff
@@ -82,26 +93,67 @@ public DependencyResult Process()
                     {
                         XmlAttribute include = attributes["Include"];
                         XmlAttribute version = attributes["Version"];
+                        XmlAttribute versionOverride = attributes["VersionOverride"];
```

Comment:
This change is making sure we are checking if the package was present in props file and if yes we are getting the version correctly. Also the change is adding support for VersionOverride tag if that is enabled. The change is same as above for ReferenceResolver.